.github/workflows/main.yml

```diff
@@ -14,3 +14,54 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build
         run: ./build.sh
+  upload:
+    needs: build
+    if: startsWith(github.ref, 'refs/tags')
+    runs-on: ubuntu-latest
+    env:
+      name: jburel/omero-web
+    steps:
+      - name: Get other tags
+        id: gettags
+        uses: jupyterhub/action-major-minor-tag-calculator@v2.0.0
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          prefix: "${{ env.name}}:"
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v2
+        with:
+          tags: ${{ join(fromJson(steps.gettags.outputs.tags)) }}
+          push: true
+  uploadstandalone:
```

Comment:
Could an environment variable: `IMAGE: ["", "standalone"]` make this impler?

Reply:
it is not working yet!

Reply:
@joshmoore i don't think so since we will need to check for parent, tag and introduce several ``if else``